packages/protocol/test/stability/exchange.ts

```diff
@@ -334,6 +392,22 @@ contract('Exchange', (accounts: string[]) => {
         assertEqualBN(actualGoldBalance, expectedGoldBalance)
       })
 
+      it('should work even if buckets need updating', async () => {
+        await fundReserve()
+        await timeTravel(updateFrequency, web3)
+
+        await exchange.exchange(
+          goldTokenAmount,
+          expectedStableBalance.integerValue(BigNumber.ROUND_FLOOR),
+          true,
+          {
+            from: user,
+          }
+        )
+        const newStableBalance = await stableToken.balanceOf(user)
+        assertEqualBN(newStableBalance, expectedStableBalance)
```

Comment:
Can you verify the buckets updated after this? Wouldn't the expected balance be different after a bucket update?